docker-compose.yml

```diff
@@ -37,4 +36,8 @@ services:
       - IRISNAMESPACE=USER
     volumes:
       - ./:/home/irisowner/zpm/
-      - ./tests/sandbox/:/home/irisowner/sandbox/
\ No newline at end of file
+      - ./tests/sandbox/:/home/irisowner/sandbox/
+      - ./tests/sandbox/fhirpro/:/fhirpro/
```

Comment:
I'm not seeing a 1tests/sandbox/fhirpro/` folder. Is it supposed to be created on first run? If so, shall we add that to .gitignore?

Reply:
This is debug code, should kill it.